support/java/Runtime.java

```diff
@@ -96,9 +99,28 @@ protected List getFieldOrder() {
     public static void initialize(String library) {
         System.setProperty("jna.encoding", "utf8");
 
+        String tmp = System.getProperty("java.io.tmpdir");
+        String assemblyPath = Utilities.combinePath(tmp, library);
+        File assemblyFile = new File(assemblyPath + ".dll");
+
+        if (!assemblyFile.exists()) {
+            String resourcePath = "/assemblies/" + library + ".dll";
+            InputStream stream = Runtime.class.getResourceAsStream(resourcePath);
+            if (stream == null) {
+                pendingException.set(new RuntimeException("Unable to locate " + resourcePath + " within jar file!"));
```

Comment:
`pendingException` is for pending managed exceptions. Here we should be able to just throw a regular Java exception.